src/script/common/c_content.cpp

```diff
@@ -2079,7 +2076,7 @@ static bool push_json_value_helper(lua_State *L, const Json::Value &value,
 			lua_pushnumber(L, value.asDouble());
 			break;
 		case Json::stringValue: {
-#ifdef HAVE_JSON_STRING
+#if JSONCPP_VERSION_HEXA >= 0x01000000 /* 1.0.0 */
```

Comment:
not 1.9.0?

Reply:
1.0.0 has `Value::toString()` https://github.com/open-source-parsers/jsoncpp/blob/1.0.0/include/json/value.h#L260

Reply:
We can remove the `#if` then as this is our minimum version :)